packages/graphql-client/package.json

```diff
@@ -34,5 +34,8 @@
     "@accounts/types": "^0.25.0",
     "graphql-tag": "^2.10.0",
     "tslib": "1.10.0"
+  },
+  "peerDependencies": {
+    "graphql": "^0.11.0 || ^0.12.0 || ^0.13.0 || ^14.0.0"
```

Comment:
also graphql-client uses graphql package (obviously 😅) but doesn't declare it as a dependency.
Added to peer dependencies (versions are compatible with apollo-client 2&3)

Reply:
@nsine good catch 😁 does it make sense to also add it as devDependency?
Also is it compatible with graphql 15?

Reply:
it shouldn't be in `devDependencies` as it's required in actual client code to work, not only while development. When choosing `dependencies` or `peerDependencies`: i found that many libraries require it as peer dep. to avoid multiple packages with different versions + there is an explanation from the creator why it's better to use peerDependencies:
https://leebyron.com/graphql-js-preparing-for-v14/#duplicate-installs-and-peer-dependencies

Regarding v15 - i'm not sure to be honest 😅We don't use it yet on our project because there are some open issues with v15 + tooling packages (apollo-link, apollo-toolkit afaik). Plus even apollo client doesn't allow to use v15 yet https://github.com/apollographql/apollo-client/blob/master/package.json#L58 So i would go with <=14 for now

Reply:
Ok perfect then 👍